bedrock/base/templates/macros-protocol.html

```diff
@@ -170,8 +170,8 @@
 
 
 {# Card: https://protocol.mozilla.org/patterns/molecules/card.html #}
-{% macro card(title, ga_title, image_url, link_url, desc=None, meta=None, cta=None, tag_label=None, media_icon=None, class=None, heading_level=2, aspect_ratio='mzp-has-aspect-3-2', include_highres_image=False, highres_image_url=None, youtube_id=None, data_modal_id=None) -%}
-<section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if youtube_id %}mzp-has-video has-video-embed{% endif %}" {% if data_modal_id %}data_modal_id="{{ data_modal_id }}"{% endif %}>
+{% macro card(title, ga_title, image_url, link_url, desc=None, meta=None, cta=None, tag_label=None, media_icon=None, class=None, heading_level=2, aspect_ratio='mzp-has-aspect-3-2', include_highres_image=False, highres_image_url=None, youtube_id=None, attributes=None) -%}
+<section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if youtube_id %}mzp-has-video has-video-embed{% endif %}" {% if attributes %}{{attributes}}{% endif %}>
```

Comment:
Created the `attributes` attribute to be used whenever we want to add a new sort of attribute to a card component. It can be used more universally than creating a one-off attribute. 
i.e. instead of creating a new attribute called `data_modal_id` and only using it in one HTML template where it's needed, the `attributes` attribute can be anything we want it to be, mainly because it has an inclusive name too.

Reply:
We should add this to the docs https://github.com/mozilla/bedrock/blob/master/docs/coding.rst#L894